src/cms/previewTemplate.js

```diff
@@ -27,8 +37,8 @@ var PreviewTemplate = ({ entry }) => {
   return (
     <body>
       <main>
-        <article className="rule-content">
-          <h1>{title}</h1>
+        <article className="rule-content editor-preview">
+          <h1 style={{ fontSize: '2.25rem' }}>{title}</h1>
```

Comment:
Should we be hardcoding the fontsize here or should it be css?

Reply:
It should be CSS - I'll fix it 👍🏻